examples/list_campaigns/main.bal

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerinax/mailchimp.marketing as mailchimp;
+
+configurable string username = ?;
+configurable string mailchimpApiKey = ?;
+
+public function main() returns error? {
+    mailchimp:Client mailchimpClient = check new ({
+        auth: {
+            username,
+            password: mailchimpApiKey
+        }
+    });
+
```

Comment:
Create the client globally, instead of creating it inside the `main` function. Do the same for the other example.

```suggestion
mailchimp:Client mailchimpClient = check new ({
	auth: {
		username,
		password: mailchimpApiKey
	}
});

public function main() returns error? {
```